packages/visual-editor/src/components/RootRenderer/DNDProvider.tsx

```diff
@@ -42,12 +44,17 @@ export const DNDProvider = ({ children }: Props) => {
     });
   };
 
+  const beforeCapture: OnBeforeCaptureResponder = () => {
+    setDraggingOnCanvas(true);
```

Comment:
adding another event handler might affect the tests on our web app's side and might require an update on that side

Reply:
these changes aren't mine and are unrelated to this PR. Not sure why they are there, I'll look into it.

Reply:
I rebased my commits on development again and these unreleated commits are gone now